packages/site/src/data/vue/communities.ts

```diff
@@ -62,7 +62,7 @@ export const communities: Community<typeof communityTags[number]>[] = [
 		name: "Vuejs Amsterdam",
 		description: "One of the largest Vue conferences in the world",
 		image:
-			"https://img2.storyblok.com/277x0/filters:format(png)/f/84560/2388x414/23d8eb4b8d/vue-amsterdam-with-name.png",
+			"https://tinyurl.com/ufuj2rv4",
```

Comment:
Please remove this because you already committed these changes in your last PR.

Make sure to merge the latest changes from main.